src/apphosting/githubConnections.ts

```diff
@@ -149,9 +150,10 @@ async function createFullyInstalledConnection(
   location: string,
   connectionId: string,
   oauthConn: devConnect.Connection,
+  withNewInstallation = false,
 ): Promise<devConnect.Connection> {
   let conn = await createConnection(projectId, location, connectionId, {
-    appInstallationId: oauthConn.githubConfig?.appInstallationId,
+    appInstallationId: withNewInstallation ? undefined : oauthConn.githubConfig?.appInstallationId,
```

Comment:
can you add a param comment to the docstring on when you would want to set this and what the default behavior is?